README.md

```diff
@@ -3,65 +3,93 @@
+> this adds a new remote named **upstream**.  
 
-  ```sh
-  $ git remote -v
-  origin    https://github.com/YOUR_USERNAME/wiki.git (fetch)
-  origin    https://github.com/YOUR_USERNAME/wiki.git (push)
-  upstream  https://github.com/FreeCodeCamp/wiki.git (fetch)
-  upstream  https://github.com/FreeCodeCamp/wiki.git (push)
-  ```
+You may see the changes with...
 
-4. Sync the fork when needed: After another PR has been merged, you will need to fetch the changes and rebase your work.
+> ```sh
+$ git remote -v
+origin    https://github.com/YOUR_USERNAME/wiki.git (fetch)
+origin    https://github.com/YOUR_USERNAME/wiki.git (push)
+upstream  https://github.com/FreeCodeCamp/wiki.git (fetch)
+upstream  https://github.com/FreeCodeCamp/wiki.git (push)
+```
 
-  ```sh
-  # fetch all remote repos and delete any deleted remote branches
-  $ git fetch --all --prune
+### 4. **Sync it** :recycle:
+Always keep your local copy of repository updated with the original repository.  
+Before making any changes and/or in an appropriate interval, run the following commands _carefully_ to update your local repository.
 
-  # switch to `master` branch
-  $ git checkout master
+```sh
+# fetch all remote repos and delete any deleted remote branches
+$ git fetch --all --prune 
 
-  # reset local `master` branch to match `upstream` repo `master` branch
-  $ git reset --hard upstream/master
+# switch to `master` branch
+$ git checkout master 
 
-  # push changes to your forked wiki repo
-  $ git push origin master
-  ```
+# reset local `master` branch to match `upstream` repo's `master` branch
+$ git reset --hard upstream/master 
 
-5. Once you have completed these steps, you can start contributing by checking our [issues](https://github.com/FreeCodeCamp/wiki/issues) and creating new pull requests.
+# push changes to your forked wiki repo
+$ git push origin master 
+```
 
-## Contribution Guides
+### 5. Ready Steady Go... :turtle: :rabbit2: 
+Once you have completed these steps, you are ready to start contributing by checking our [![Help Wanted Issues](https://img.shields.io/badge/Wiki-Help_Wanted_Issues-159818.svg?style=flat-square&maxAge=2592000)](https://github.com/FreeCodeCamp/wiki/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22 "Help Wanted Issues") and creating *new shiny* :sparkles: [pull requests](https://github.com/FreeCodeCamp/wiki/pulls).
 
-We currently have a couple of guides to help you contribute, via the browser, command line or desktop application. You can find this and many more guides and information on the [Wiki Central Page](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Wiki).
+## Help Contributing Guides
+We also have guides in our Wiki to assist you in your contributions. You can find this and many more guides, tutorials and other information in the [Wiki Central](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Wiki)  Page such as  
+
+> Contributing Using   
```

Comment:
I didn't want to point to the actual guides directly becauase we have changed the links a couple of time, this would just add one more place where it would need to be changed and these instructions and file overall should not have the need to be updated given that the purpose is to provide the instructions to fork, clone and add the remote for sync.

The link provided on like 81 is where the guides will always be thus making this file one that shouldn't need update once we finalize it.